modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
```

Comment:
On my first pass, I missed this. According to the mod docs guide, procedures can only have one heading per module.